config/opensearch_dashboards.yml

```diff
@@ -151,16 +151,15 @@
 # ]
 #vis_type_timeline.graphiteBlockedIPs: []
 
-# full version customized logo URL
-# opensearchDashboards.branding.fullLogoUrl: ""
-
-# smaller version customized logo URL
-# opensearchDashboards.branding.logoUrl: ""
-
-# customized loading logo URL
-# opensearchDashboards.branding.loadingLogoUrl: ""
-
-# custom application title
-# opensearchDashboards.branding.title: ""
-
-
+# opensearchDashboards.branding:
+  # logo:
+    # default: ""
+    # darkmode: ""
```

Comment:
Think technically the application treats dark mode as two words so it would be darkMode.

Also did we want to abandon the "url" aspect of it. i think it might make it easier to understand it is defaultUrl and darkModeUrl.

What do you think?

Reply:
Thx for the advice, yea I think we can add url to make it more explicit to the users.